src/WebJobs.Script.Grpc/Server/GrpcServer.cs

```diff
@@ -16,7 +16,7 @@ public class GrpcServer : IRpcServer, IDisposable
         private int _shutdown = 0;
         private Server _server;
         private bool _disposed = false;
-        public const int MaxMessageLengthBytes = 128 * 1024 * 1024;
+        public const int MaxMessageLengthBytes = int.MaxValue;
```

Comment:
Could you change this to int.MaxValue too?
https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Workers/Rpc/RpcWorkerConstants.cs#L31

Reply:
Is there a chance we could start run into timeout issues the payload size is close to this new max ?

Reply:
good catch @mhoeger. Fixed it.

@yojagad - there is a chance that app might run into out-of-memory exception which is by design.